NuRadioMC/test/atmospheric_Aeff/1e18eV/T03check_output.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+import numpy as np
+import h5py
+from NuRadioReco.utilities import units
+from NuRadioMC.utilities.Veff import get_triggered
+import sys
+import os
+
+###########################
+# Reference values from previous run, have to be updated, if code changes
+###########################
+
+# the event generation used to have a fixed seed. The Askaryan model is Alvarez2000,
+# so as a result the effective area should have no statistical scatter. However,
+# Proposal gives a different result for the same seed depending on the system,
+# so now we don't fix the seed and check that the calculated effective area is
+# within a number of standard of the previous calculation.
+Aeff_mean = 0.406902 * units.km2
+Aeff_sigma = 0.045211 * units.km2
```

Comment:
can you add as a comment the info how you obtained this number? E.g. how often did you run the script (to get a sense of the statistical precision of the sigma value). 
Actually, how do you obtain it? You did not override the seed. Shouldn't you get always the same result then?

Reply:
I don't override the seed because as I explain on the comment, the results obtained with Proposal depend on the system. I just ran the example for 5e3 events and took the Poissonian estimator for the uncertainty.

Reply:
And a fixed seed using Proposal doesn't mean anything if the input properties of the muons are different - the results are going to be different each time if e.g. the positions change.

Reply:
If we want a more realistic number I would need to run a large simulation, but keep in mind that our test results might be biased because we are restricted to low statistics (a few thousand events).

Reply:
clever idea. So you just took sqrt(N_triggered)*Aeff  as the uncertainty?

Reply:
Right, with the sqrt() in the denominator. There are some subtleties because it's not the same estimator (the sample standard deviation does not make any assumption on the distribution) and that can give problems for low statistics, but it seems to work for 5e3 events. I'm going to remove pair production and see if I can get more statistics easily.